LFS-639 / LFS-684: Add note field for Demographics fields: Reason for Dx,  Mutation-Pathogenicity, Date of Dx, Genetic Diagnosis

OrdiNeu opened the pull request.

OrdiNeu committed:
LFS-639 / LFS-684: Add note field for Demographics fields: Reason for Dx,
Mutation-Pathogenicity, Date of Dx, Genetic Diagnosis

sdumitriu reviewed (commented):
Doesn't the closely-related "Mutation-Function" question also need notes?

OrdiNeu committed:
LFS-639 / LFS-684: Add note field for Demographics fields: Reason for
Dx, Mutation-Pathogenicity, Date of Dx, Genetic Diagnosis

Also add note for Mutation-Function

OrdiNeu commented:
> Doesn't the closely-related "Mutation-Function" question also need notes? 

I didn't see any instances when I was preparing LFS data, but I've added it for `2a3e279a`.

sashaandjic commented:
The following 4 notes fields are added. 
![image](https://user-images.githubusercontent.com/23243244/98195052-d7698a80-1eee-11eb-9a0b-698a3cfe4126.png)
![image](https://user-images.githubusercontent.com/23243244/98195068-de909880-1eee-11eb-959a-af8fc950eb7b.png)
![image](https://user-images.githubusercontent.com/23243244/98195076-e5b7a680-1eee-11eb-8935-78eb97b03d6b.png)
![image](https://user-images.githubusercontent.com/23243244/98195108-f1a36880-1eee-11eb-8f58-965d8b3b12f3.png)

sashaandjic reviewed (approved):
tested
